mantisshrimp/datasets/pets/transforms.py

```diff
@@ -0,0 +1,59 @@
+__all__ = [
+    "aug_tfms_albumentations_pets"
+]
+
+from mantisshrimp.imports import *
+from mantisshrimp.core import *
+from mantisshrimp.transforms.albu_transform import *
+from mantisshrimp.utils.utils import *
+
+import albumentations as A
+
+
+
+def aug_tfms_albumentations_pets(train=True):
```

Comment:
I think we can remove `aug`, since if `train=False` no augmentation transforms are used.

What do you think of the idea of having two functions? `train_albumentations_tfms_pets` and `valid_albumentations_tfms_pets`? That also eliminates the need of `if` inside the function.